indico/modules/events/persons/__init__.py

```diff
@@ -32,3 +32,17 @@
 def _sidemenu_items(sender, event, **kwargs):
     if event.can_manage(session.user):
         return SideMenuItem('persons', _('Roles'), url_for('persons.person_list', event), section='reports')
+
+
+@signals.get_placeholders.connect_via('event-persons-email')
+def _get_placeholders(sender, person, event, register_link=True, **kwargs):
```

Comment:
Well I tried it with false more times but then there was some problem with render method or something (it was complaining about arguments), and in 2 cases we need it true and in only one its false so I think this logic makes sense ;)